File: JavaScript/1041.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
const pegarValores = (line) => line.split(" ").map(a => +a);
let [x,y] = pegarValores(lines.shift())
if(x == 0 && y == 0){
    console.log("Origem")
}else if(x == 0){
    console.log("Eixo Y")
}else if(y == 0){
    console.log("Eixo X")
}else if(x > 0 && y > 0){
    console.log("Q1")
}else if(x > 0 && y < 0){
    console.log("Q4")
}else if(x < 0 && y > 0){
    console.log("Q2")
}else if(x < 0 && y < 0){
    console.log("Q3")
}

File: Python/1041.py
lista = (input().split(" "))
x, y = lista
x = float(x)
y = float(y)

if(x == 0 and y == 0):
    print("Origem")
elif(x == 0):
    print("Eixo Y")
elif(y == 0):
    print("Eixo X")
elif(x > 0 and y > 0):
    print("Q1")
elif(x > 0 and y < 0):
    print("Q4")
elif(x < 0 and y > 0):
    print("Q2")
elif(x < 0 and y < 0):
    print("Q3")
